relay-server/src/actors/project_cache.rs

```diff
@@ -378,17 +378,46 @@ struct UpdateProjectState {
     no_cache: bool,
 }
 
+/// Holds the addresses of all services required for [`ProjectCache`].
+#[derive(Debug, Clone)]
+pub struct ProjectCacheContext {
```

Comment:
nit(naming): I would rename this to `Services` because it is a better description of what's inside. Other mods can then call `project_cache::Services::new()`.

Or even better, call it `AddressBook` :)

Reply:
Even though I like the `AddressBook`, but still renamed it to `Services` for now 😄